oscaralanpierce/skyrim_inventory_management_frontend
# Remove notes from aggregate list items in UI

## Context

[**Remove notes from aggregate list items in UI**](https://trello.com/c/LH7p9Z0G/293-remove-notes-from-aggregate-list-items-in-ui)

in danascheider/skyrim_inventory_management#164, we changed aggregation of shopping and inventory list items such that aggregate list items no longer have `notes` values. This being the case, it doesn't make sense for them to display an always-blank `notes` field in the UI. We wanted to change it so aggregate (non-editable) list items don't have `notes` in the UI.

## Changes

* Remove `Notes` field from aggregate shopping list items
* Small styling tweak to make the bottom margin of the shopping list item details section appear the same as the top margin
* Update tests and snapshots
* Update test data

## Required Tasks


## Manual Test Cases

To perform these tests, you will need a regular shopping list and an aggregate shopping list with at least one shopping list item. You'll need to be logged in, on the shopping lists page, and have the game those lists belong to selected.

1. Expand a shopping list item on the aggregate list
2. See that there is no `Notes` field
3. Expand a shopping list item on a regular list
4. See that there is still a `Notes` field

## Screenshots and GIFs

### 320px

<img width="320" alt="ShoppingLists-320" src="https://user-images.githubusercontent.com/5115928/231098259-108ebe4a-8456-4175-890e-eb1d88201da2.png">

### 481px

<img width="482" alt="ShoppingLists-481" src="https://user-images.githubusercontent.com/5115928/231098283-8dbd4e33-bba7-47c4-aaff-2a3c7a2f8b64.png">

### 601px

<img width="601" alt="ShoppingLists-601" src="https://user-images.githubusercontent.com/5115928/231098317-b0481c42-172e-407d-b102-73b179a226e6.png">

### 769px

<img width="770" alt="ShoppingLists-769" src="https://user-images.githubusercontent.com/5115928/231098356-4382fa06-8bc4-4bfd-96ad-23a0673bbc2a.png">

### 1025px

<img width="1027" alt="ShoppingLists-1025" src="https://user-images.githubusercontent.com/5115928/231098394-88db4253-75af-46dc-a6af-4ee338802470.png">

### 1201px

<img width="1201" alt="ShoppingLists-1201" src="https://user-images.githubusercontent.com/5115928/231098425-09caff90-f8a5-4abf-b323-4b699b610e2c.png">

### 1405px

<img width="1201" alt="ShoppingLists-1201" src="https://user-images.githubusercontent.com/5115928/231098461-6401ea24-055c-4917-9beb-749f94022a84.png">

### Large Desktop

<img width="1920" alt="ShoppingLists-Full" src="https://user-images.githubusercontent.com/5115928/231098512-7d585fc8-7fa4-4bd4-aa50-c241aaa5d62f.png">
